### PR TITLE
Secret selection from modal

### PR DESCRIPTION
**Pull request by cristineguadelupe**
--- WIP ---

### CODE REVIEW

**Review by josevalim — COMMENTED** [review]
LGTM, waiting for @jonatanklosko's approval. :)

**Review by jonatanklosko — APPROVED** [review]
Looks good, feel free to merge as soon as the Livebook part is ready :)

**Pull request merged**
